DisplayInfo/CMakeLists.txt

```diff
@@ -75,7 +75,8 @@ if (USE_DEVICESETTINGS)
     target_link_libraries(${MODULE_NAME} PRIVATE
         ${NAMESPACE}Plugins::${NAMESPACE}Plugins
         ${IARMBUS_LIBRARIES}
-        ${DS_LIBRARIES})
+        ${DS_LIBRARIES}
+	displayinfo-soc)
```

Comment:
Shouldn't the dependency be to virtual/vendor-displayinfo-soc instead of directly to displayinfo-soc.